.evergreen/run-orchestration.sh

```diff
@@ -95,3 +98,8 @@ cat <<EOT >> $DRIVERS_TOOLS/results.json
 ]}
 
 EOT
+
+# Set the requireApiVersion parameter
+if [ ! -z "$REQUIRE_API_VERSION" ]; then
+  mongo $URI $MONGO_ORCHESTRATION_HOME/require-api-version.js
```

Comment:
If this is accidentally done against a server that doesn't support require api version, will it cause m-o to fail (hoping yes)

Reply:
I was hoping it would, but it looks like `mongo` returns a 0 exit code even when the command fails:
```
$ mongo .evergreen/orchestration/require-api-version.js 
MongoDB shell version v4.2.11
connecting to: mongodb://127.0.0.1:27017/?compressors=disabled&gssapiServiceName=mongodb
Implicit session: session { "id" : UUID("8610a0d0-8064-4066-b2d5-29601a33025a") }
MongoDB server version: 4.2.11

$ echo $?
0
```

However, this also holds true for using `eval` when the command fails:

```
$ mongo --eval 'db.adminCommand({ "setParameter": 1, "requireApiVersion": 1 });'
MongoDB shell version v4.2.11
connecting to: mongodb://127.0.0.1:27017/?compressors=disabled&gssapiServiceName=mongodb
Implicit session: session { "id" : UUID("c43b0af6-0c0e-4ac9-96ae-7c1d429d6992") }
MongoDB server version: 4.2.11
{
	"ok" : 0,
	"errmsg" : "attempted to set unrecognized parameter [requireApiVersion], use help:true to see options "
}

$ echo $?
0
```

The only way I could come up with is to combine `--eval` with parsing the response to make sure the command succeeded, but I don't think this is worth it.

If there are no objections, I'd merge this down hoping that we can have versioned API support in m-o sooner than later.